docs/getting_started/access_to_ez_enterprise.md

```diff
@@ -0,0 +1,58 @@
+# Access to eZ Enterprise
+
+!!! enterprise
+
+    ## Setting up authentication tokens for eZ Enterprise
+
+    eZ Enterprise subscribers have access to commercial packages at [updates.ez.no/bul/](https://updates.ez.no/bul/).
+    The site is password-protected. You will need to set up authentication tokens to get access to them.
+
+    Log in to your service portal on [support.ez.no](https://support.ez.no) and look for the following on the *"Maintenance and Support agreement details"* screen:
+
+    ![Authentication token](img/Using_Composer_Auth_token.png)
+
+    1. Click "Create token" (this requires the "Portal administrator" access level).
+    2. Fill in a label describing the use of the token. This will allow you to revoke access later.
+        - For example, if you need to provide access to updates to a third party, a good example would be "53-upgrade-project-by-partner-x"
```

Comment:
For example, if you want to provide access to a limited number of updates to a third party. 

a good example would be "53-upgrade-project-by-partner-x" - remove